Cartfile

```diff
@@ -2,7 +2,7 @@ binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.3.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 22.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" ~> 6.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" == 7.1.2
-github "mapbox/mapbox-directions-swift" ~> 1.0
+github "mapbox/mapbox-directions-swift" "vk/478-intersection-geometry"
```

Comment:
Remember to change this dependency to `~> 1.2` for both Carthage and CocoaPods once mapbox/mapbox-directions-swift#490 lands, or change it back to `~> 1.0` and ensure that the SDK can handle an older version of MapboxDirections gracefully.

Reply:
Yes, will update it after https://github.com/mapbox/mapbox-directions-swift/pull/490 lands.